tableauserverclient/models/data_alert_item.py

```diff
@@ -0,0 +1,190 @@
+import xml.etree.ElementTree as ET
+
+from .property_decorators import property_not_empty
+from .user_item import UserItem
+from .view_item import ViewItem
+
+
+class DataAlertItem(object):
+    def __init__(self):
+        self._id = None
+        self._subject = None
+        self._creatorId = None
+        self._createdAt = None
+        self._updatedAt = None
+        self._frequency = None
+        self._public = None
+        self._owner_id = None
+        self._owner_name = None
+        self._view_id = None
+        self._view_name = None
+        self._workbook_id = None
+        self._workbook_name = None
+        self._project_id = None
+        self._project_name = None
+        self._recipients = None
+
+    def __repr__(self):
+        return "<Data Alert {id} subject={subject} frequency={frequency} \
+                public={public}>".format(**self.__dict__)
+
+    @property
+    def id(self):
+        return self._id
+
+    @property
+    def subject(self):
+        return self._subject
+
+    @subject.setter
+    @property_not_empty
+    def subject(self, value):
+        self._subject = value
+
+    @property
+    def frequency(self):
+        return self._frequency
+
+    @frequency.setter
+    @property_not_empty
```

Comment:
I think it would be really nice to have a frequency enum that lists out the 5 valid values. You can also use the enum property decorator to ensure that a user passes in a valid frequency value. Similar to how it's done in schedule_item.py

Reply:
Created enum and used decorator to validate values.